framework/doc/content/source/actions/AdaptivityAction.md

```diff
@@ -1,14 +1 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
-# AdaptivityAction
-
-!alert construction title=Undocumented Class
-The AdaptivityAction has not been documented. The content contained on this page includes the
-typical automatic documentation associated with an Action; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
-
-!syntax description /Executioner/Adaptivity/AdaptivityAction
-
-!syntax parameters /Executioner/Adaptivity/AdaptivityAction
-
-!bibtex bibliography
+!template load file=stubs/moose_action.md.template name=AdaptivityAction syntax=/Executioner/Adaptivity/AdaptivityAction
```

Comment:
This adds another step to turning an undocumented object into a documented object :-(. Previously one could just start from the stub, but now that structure is gone.

Reply:
I know this involves running the server and copy and pasting, but does this help alleviate the problem you have with this?

My idea is that you generate, serve, and go to the page. You see the following, which you copy and paste and start working. Is that no good?

https://mooseframework.inl.gov/docs/PRs/13692/site/source/auxkernels/MaterialRealTensorValueAux.html